src/Homework4/Actions.java

```diff
@@ -0,0 +1,10 @@
+package Homework4;
+
+public interface Actions {
+
+    void createPlant( String name, String type, String nativeRegion, boolean isWatered );
```

Comment:
And again formating :) Don't use spaces when opening and closing brackets, just createPlant(String name); that's all

Reply:
It's interesting that ctr+alt+L adding spaces in some brackets. Deleted them.